todos.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>todos</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div>
+       <H1>todos</H1>
```

Comment:
replace H1 by h1
This can be misleading because only custom tags will start from big letter